cmake/BuildOptionDefaults.cmake

```diff
@@ -24,7 +24,8 @@ set(NRN_ENABLE_MOD_COMPATIBILITY_DEFAULT OFF)
 set(NRN_ENABLE_REL_RPATH_DEFAULT OFF)
 set(NRN_ENABLE_INTERNAL_READLINE_DEFAULT OFF)
 
-set(NRN_MODULE_INSTALL_OPTIONS_DEFAULT "--home=${CMAKE_INSTALL_PREFIX}")
+# Some distributions may set the prefix. To avoid errors, unset it
+set(NRN_MODULE_INSTALL_OPTIONS_DEFAULT "--prefix= --home=${CMAKE_INSTALL_PREFIX}")
```

Comment:
I chose --home originally without  understanding the distinct purposes of --home and --prefix. I still don't. Even after reading a few documents.  But this looks fine to me.

Reply:
Same here...and after reading Michael comment I wouldn’t try to read about it 😀

But as Fernando was debugging this today, I believe this should be fine. If CI passes, good merge this!